app/plugins/prometheus.md

```diff
@@ -0,0 +1,171 @@
+---
+id: page-plugin
+title: Plugins - Prometheus
+header_title: Prometheus
+header_icon: https://konghq.com/wp-content/uploads/2018/06/prometheus.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Usage
+    items:
+      - label: Log Format
+description: |
+  This plugin exposes metrics in [Prometheus Exposition format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md). It allows you to integrate with your existing monitoring and alerting system.
+
+params:
+  name: prometheus
```

Comment:
This plugin can be applied on specific APIs/routes/services to collect stats related to specific routes/services only.
So could we update this to:
```yaml
params:
  name: prometheus
  api_id: true
  service_id: true
  route_id: true
```

Reply:
Done